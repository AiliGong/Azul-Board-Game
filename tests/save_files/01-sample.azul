RYYURBBBBLLLRRUURYBLRYYURBBBBLLLRRUURYBLRYYURBBBBLLLRRUURYBLRYYURBBBBLLLRRUURYBLRYYURBBBBLLLRRUURYBL
A
B
turn 2 B 3
turn 3 L 3
