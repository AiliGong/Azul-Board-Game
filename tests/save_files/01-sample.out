Factories:
0: F R B
1: R Y Y U
2:
3:
4: R R U U
5: R Y B L

Score for A: 0
Mosaic for A:
1:         . || . . . . . 
2:       . . || . . . . . 
3:     B B B || . . . . . 
4:   . . . . || . . . . . 
5: . . . . . || . . . . . 
broken:

Score for B: 0
Mosaic for B:
1:         . || . . . . . 
2:       . . || . . . . . 
3:     L L L || . . . . . 
4:   . . . . || . . . . . 
5: . . . . . || . . . . . 
broken:
